src/frontend/js/modules/session.js

```diff
@@ -93,17 +95,47 @@
   })
 
   // TODO (esn-frontend-common-libs#51): Write tests for the new changes (https://github.com/OpenPaaS-Suite/esn-frontend-common-libs/pull/48)
-  .controller('sessionInitESNController', function($scope, esnTemplate, sessionFactory) {
+  .controller('sessionInitESNController', function($scope, $log, esnTemplate, sessionFactory, userAPI, httpConfigurer) {
+    const onLogin = data => {
+      $log.debug('User logged in', data);
```

Comment:
For the security issue, should we print out the user's login data?

Reply:
True